auth/grpctransport/directpath.go

```diff
@@ -22,7 +22,7 @@ import (
 	"strings"
 
 	"cloud.google.com/go/auth"
-	"cloud.google.com/go/compute/metadata"
+	"cloud.google.com/go/internal/compute"
```

Comment:
This import should be `cloud.google.com/go/auth/internal/compute`.

Reply:
done, thanks!